### DISCUSSION TITLE
some about import excel that contain picture

### DISCUSSION THREAD

**Discussion by Justlov** [django-import-export/django-import-export#1290] [Q&A]
![image](https://user-images.githubusercontent.com/43231862/117436792-f6aaa880-af61-11eb-8a5d-8e6ad37eb754.png)

**Comment by andrewgy8** [comment]
Is there a question here @Justlov ? Or is this an issue which should be opened in the issues section?

**Reply by Justlov** [reply]
i  don't know how to import the exect  with pictuer  into database...  I read the execl  with pandas  but  can.t read the picture. why.....

**Reply by andrewgy8** [reply]
Do you have any code/examples to show? Its hard to know whats going wrong without something to work with.

**Reply by Justlov** [reply]
![image](https://user-images.githubusercontent.com/43231862/117448278-21502d80-af71-11eb-9bbf-80628c50e603.png)
like this  .  i want to import this execl  into mysql   database .

**Reply by Justlov** [reply]
read the execl  can't read image .....
